analysed-packages/libpng/version-1.6.39/libpng-1.6.39-OSS-disclosure.txt

```diff
@@ -0,0 +1,2338 @@
+Copyright (c) 1998-2002,2004,2006-2014,2016,2018 Glenn Randers-Pehrson
+Copyright (c) 1998-2002,2004,2006-2014,2016 Glenn Randers-Pehrson
+Copyright (c) 1998-2002,2004,2006-2013,2018 Glenn Randers-Pehrson
+Copyright (c) 1998-2002,2004,2006-2013 Glenn Randers-Pehrson
+Copyright (c) 1998-2000 Glenn Randers-Pehrson
+Copyright (c) 1996-1997 Andreas Dilger
+Copyright (c) 1996, 1997 Andreas Dilger
+Copyright (c) 1996 Massachusetts Institute of Technology.
+Copyright (c) 1995-2022 The PNG Reference Library Authors: * Andreas Dilger *
+Cosmin Truta * Dave Martindale * Eric S. Raymond * Gilles Vollant * Glenn
+Randers-Pehrson * Greg Roelofs * Guy Eric Schalnat * James Yu * John Bowler *
+Kevin Bracey * Magnus Holmgren * Mandar Sahastrabuddhe * Mans Rullgard * Matt
+Sarett * Mike Klein * Pascal Massimino * Paul Schmidt * Qiang Zhou * Sam Bushell
+* Samuel Williams * Simon-Pierre Cadieux * Tim Wegner * Tom Lane * Tom Tanner *
+Vadim Barkov * Willem van Schaik * Zhijie Liang * Arm Holdings - Richard
+Townsend * Google Inc. - Dan Field - Leon Scroggins III- Matt Sarett - Mike Klein - Sami Boukortt
+Copyright (c) 1995-1996 Guy Eric Schalnat, Group 42, Inc.
+Copyright (c) 1995, 1996 Guy Eric Schalnat, Group 42, Inc.
+Copyright (c) 1995, 1996 Frank J. T. Wojcik December 18, 1995 & January 20, 1996
+Copyright (C) 2022 Cosmin Truta
+Copyright (C) 2021 Kirk Roerig
+Copyright (C) 2021 Gabor Kertesz
+Copyright (C) 2021 Free Software Foundation, Inc.
+Copyright (C) 2020-2022 Cosmin Truta
+Copyright (C) 2019-2022 Cosmin Truta
+Copyright (C) 2016 Glenn Randers-Pehrson Written by Roger Leigh, 2016
+Copyright (C) 2014 Glenn Randers-Pehrson and Andrey A. Chernov
+Copyright (C) 2014 Free Software Foundation, Inc.
+Copyright (C) 2012 Glenn Randers-Pehrson and Christopher M. Wheeler
+Copyright (C) 2011-2021 Free Software Foundation, Inc.
+Copyright (C) 2011 Free Software Foundation, Inc
+Copyright (C) 2010-2019, 2021 Bootstrap Authors
+Copyright (C) 2009-2021 Free Software Foundation, Inc.
+Copyright (C) 2008, 2014 Glenn Randers-Pehrson
+Copyright (C) 2007-2009, 2014 Glenn Randers-Pehrson
+Copyright (C) 2006-2021 Free Software Foundation, Inc.
+Copyright (C) 2006,2009,2011,2014 Glenn Randers-Pehrson
+Copyright (C) 2006, 2014 Glenn Randers-Pehrson
+Copyright (C) 2004-2021 Free Software Foundation, Inc.
+Copyright (C) 2004-2019, 2021 Bootstrap Authors
+Copyright (C) 2004-2005, 2007-2009, 2011-2019, 2021-2022 Free Software
+Foundation, Inc. Written by Gary V. Vaughan, 2004
+Copyright (C) 2004-2005, 2007-2008, 2011-2019, 2021-2022 Free Software
+Foundation, Inc. Written by Gary V. Vaughan, 2004
+Copyright (C) 2004-2005, 2007, 2009, 2011-2019, 2021-2022 Free Software
+Foundation, Inc. Written by Scott James Remnant, 2004.
+Copyright (C) 2004, 2011-2019, 2021-2022 Free Software Foundation, Inc. Written
+by Scott James Remnant, 2004
+Copyright (C) 2004, 2006-2008, 2010-2014 Glenn Randers-Pehrson Contributed by
+William L. Sebok, based on makefile.linux
```

Comment:
please remove "based on makefile.linux" . Please remove this in the other files too